chore: add new enum type event (#71)
skore-io/nestjs-extensions: packages/event/README.md

# Event

Lib to send events according to the selected client

## Installation

```bash
npm i @skore-io/event
```

## PubSub

### Setup

Set environment variables in the project

```sh

> GCP_EVENTS_PROJECT_URL='https://pubsub.googleapis.com/v1/projects/skore-events-staging/topics/events:publish'

Reviewer: ```suggestion
GCP_EVENTS_PROJECT_URL='https://pubsub.googleapis.com/v1/projects/{{project_name}}/topics/events:publish'
```
O que acha? Pois podemos variar dependendo do ambiente;

Author: boaaa

GOOGLE_APPLICATION_CREDENTIALS='gcloud-service-account.json'
```

**\*gcloud-service-account.json**: json file must be in root project\*

### Usage

> import and create attributes

Reviewer: ```suggestion
1. Import and create attributes
```

Author: boa, vou add o sequencial command haha

```typescript
import {
  SendEventService,
  PubSubAttributeDto,
  PubSubActionEnum,
  PubSubTypeEventEnum,
  ClientEventNameEnum,
} from '@skore-io/event'
```

```typescript
const attributes: PubSubAttributeDto = {
  action: PubSubActionEnum.send_notification,
  source: 'workspace:yolo.ts',
  type: PubSubTypeEventEnum['io.skore.commands.integration'],
}
```

> Create instance for the send event class

Reviewer: ```suggestion
2. Create an instance of the SendEventService class
```

```typescript
    try {
      const sendEvent = new SendEventService(ClientEventNameEnum.PubSub)
```

> Call method "perform" with attributes and body

Reviewer: ```suggestion
3. Call method "perform" with attributes and body
```

```typescript
      const body = {
         test: 'yolo',
      }

      await sendEvent.perform(attributes, body)

>       this.logger.log(`New message send with succeffully!`)

Reviewer 2: ```suggestion
      this.logger.log('New message send with succeffully!')
```

Author: feitoria

>       const body = {
>          test: 'yolo',
>       }
> 
>       await sendEvent.perform(attributes, body)
> 
>       this.logger.log(`New message send with succeffully!`)
>     } catch (error) {
>       this.logger.error(error)
>     }

Reviewer: ```suggestion
    const body = { test: 'yolo' }

    await sendEvent.perform(attributes, body)
```
Acho que a parte do try/catch e dos logs fica mais à cargo do projeto que está usando a lib, não? Talvez não precise deles no README. O que acha?

Author: é realmente fica meio livre né o intuito do try era mostrar que pode ser gerado um erro do momento que é feita uma instancia ao envio do evento, mas isso relamente pode ficar sub entendido. vou mudar
